### DISCUSSION TITLE
use ImageDescription EXIF field to store film and filters data

### DISCUSSION THREAD

**Discussion by cezary-zemis** [tommi1hirvonen/ExifNotes#34] [Ideas]
EXIF does not support adding custom fields, but I would like to keep information of film and filters used. I suggest allowing putting this into "ImageDescription", e.g. the following way:
-ImageDescription="Shot on: FOMAPAN 100 Classic|Filter: Hoya HD CIR-PL 67mm|Kenko R1 67mm"

**Comment by AngelaDMerkel** [comment]
It is possible to write custom metadata and Lightroom can be configured to read them. Negative Lab Pro uses custom metadata fields in order to write the film type, stock etc. If ExifNotes conformed it would be possible to read from that into NLP in Lightroom.

**Reply by cezary-zemis** [reply]
This would be even more flexible, however it would be best to let a user to configure it according to their needs.

**Reply by AngelaDMerkel** [reply]
I personally disagree, because I think it opens the way for redundant fields which users insert in error and which cannot be properly read or carried anywhere. Most programs will not read custom metadata, so using fields other applications support would be preferred. This can be seen in cases where the manufacturer metadata is lost converting to DNG.
